app/matplotlib.ipynb

```diff
@@ -339,16 +463,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
+   "execution_count": null,
    "outputs": [],
    "source": [
-    "# write your code here\n"
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "df = pd.read_csv(\"/Users/andriysmacbookpro/PycharmProjects/Mate/data-analisys/\"\n",
+    "                 \"py-matplotlib/data/customer_data.csv\")\n",
```

Comment:
Do not use an absolute path. Because this only works on your computer. Use relative path
data/customer_data.csv